commands/device/stop-application.ts

```diff
@@ -13,11 +13,11 @@ export class StopApplicationOnDeviceCommand implements ICommand {
 		private $staticConfig: Config.IStaticConfig,
 		private $options: IOptions) { }
 
-	allowedParameters: ICommandParameter[] = [this.$stringParameter];
+	allowedParameters: ICommandParameter[] = [this.$stringParameter, this.$stringParameter];
 
 	public execute(args: string[]): IFuture<void> {
 		return (() => {
-			this.$devicesServices.initialize({ deviceId: this.$options.device, skipInferPlatform: true }).wait();
+			this.$devicesServices.initialize({ deviceId: this.$options.device, skipInferPlatform: true, platform: args[1] }).wait();
```

Comment:
This change is required because when livesyncing to ios device and android device is connected at the same time we're initializing and starting device discovery without platform. This triggers android discovery and thus throws an error. Actually initialing device service again is not correct (we've already initialized it at the begging of livesync command) and the change actually passed the specified platform to device service.